app/build.gradle

```diff
@@ -1,16 +1,16 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
-apply plugin: "org.jlleitschuh.gradle.ktlint"
+//apply plugin: 'io.gitlab.arturbosch.detekt'
+//apply plugin: "org.jlleitschuh.gradle.ktlint"
 
-import org.jlleitschuh.gradle.ktlint.reporter.ReporterType
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
     defaultConfig {
         applicationId "com.toolslab.reviewme"
         minSdkVersion 28
-        targetSdkVersion 28
+        targetSdkVersion 29
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/errysuprayogi/ReviewMe/blob/2853431aaa2d2b491e997e69b726255630c3820f/app/build.gradle#L13"></span>Not targeting the latest versions of Android; compatibility modes apply. Consider testing and updating this version. Consult the android.os.Build.VERSION_CODES javadoc for details.</td>
    </tr>
  </tbody>
</table>